main.js

```diff
@@ -1,8 +1,61 @@
 function writeToDom(domString, domId){
   var myDiv = document.getElementById(domId);
-  myDiv.innerHTML += domString;
+  myDiv.innerHTML = domString;
 }
 
```

Comment:
this writeToDom wont work if it doesnt check if the div is there first

Reply:
ok - fixed it